content/preprocessing/01-basic-qc.ipynb

```diff
@@ -34,13 +34,38 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 4,
    "metadata": {},
    "outputs": [],
    "source": [
     "adata = sc.read('../data/brain_raw.h5ad')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As an aside, you'll notice that both here and in the previous notebook we read data into python objects using some variation on `package.read*`, where `*` indicates that there is some possible stem. Developers of python data packages that we'll be using today, and many others in the ecosystem try very hard to group objects so that they are discoverable. \n",
```

Comment:
```suggestion
    "As an aside, you'll notice that both here and in the previous notebook we read data into python objects using some variation on `package.read*`, where `*` indicates that there is some possible suffix. Developers of python data packages that we'll be using today, and many others in the ecosystem try very hard to group objects so that they are discoverable. \n",
```